symfony/plugins/orangehrmAdminPlugin/modules/admin/actions/payGradeAction.class.php

```diff
@@ -58,6 +58,10 @@ public function execute($request) {
         }
         $values = array('payGradeId' => $this->payGradeId, 'payGradePermissions' => $this->payGradePermissions);
 
+        $payGrade = $this->getPayGradeService()->getPayGradeById($this->payGradeId);
+        if (!($payGrade instanceof PayGrade) && !is_null($this->payGradeId)) {
```

Comment:
why do you need to check if payGradeId is not null? isn't checking $paygrade instance of PayGrade enough?